### PR TITLE
docs: add svn post commit hook user guide

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @cd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/user_guide.md [hunk 1]
```diff
@@ -66,4 +66,51 @@ BitBucket server supports basic webhook starting with **5.4**, and in **5.10**,
...
+Login your svn server, and do the following steps to configure hooks:
+- Make sure `curl` is installed at `/usr/bin/curl` in you svn server.
+
+- Navigate to your repository’s hooks directory. This is almost always a directory cleverly named “hooks” right inside the top level of your repository:
+```
+cd /home/svn/my_repository/hooks/
+```
+
+- Create a new file called post-commit, and make it executable.
+```
```
Comment (supereagle) [inline docs/user_guide.md]:
```suggestion
```shell
$ cat <<EOF > post-commit.sh
#!/bin/sh
REPOS="\$1"
REV="\$2"
TXN_NAME="\$3"
UUID=\`svnlook uuid \$REPOS\`
/usr/bin/curl --request POST \\
              --header "Content-Type:application/json;charset=UTF-8" \\
              --header "X-Subversion-Event:Post-Commit" \\
              --data "{\"repoUUID\":\"${UUID}\", \"revision\":\"${REV}\"}" \\
              http://{cyclone-server-address}/apis/v1alpha1/tenants/{tenant}/webhook?sourceType=SCM
EOF
$ chmod 755 post-commit.sh
```

File: docs/user_guide.md [hunk 2]
```diff
@@ -66,4 +66,51 @@ BitBucket server supports basic webhook starting with **5.4**, and in **5.10**,
...
+              --header "Content-Type:application/json;charset=UTF-8" \
+              --header "X-Subversion-Event:Post-Commit" \
+              --data "{\"repoUUID\":\"${UUID}\", \"revision\":\"${REV}\"}" \
+              http://{cyclone-server-address}/apis/v1alpha1/tenants/{tenant}/webhook?sourceType=SCM
+```
+Please replace `{cyclone-server-address}` and `{tenant}` with correct value.
+
+### Create SCM type workflowtriggers
+
+Note that the `workflowtrigger.spc.scm.repo` field should be svn repository's uuid, you can get it by:
```
Comment (supereagle) [inline docs/user_guide.md]: s/svn/SVN/g

Please also fix other places.

**Comment by zhujian7** [comment]
@supereagle PTAL

**Pull request merged**